openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1136,10 +1164,10 @@ Communication with the Notification Endpoint MUST utilize TLS.
 
 ## Notification Request {#notification}
 
-The Wallet sends an HTTP POST request to the Notification Endpoint with the following parameters in the entity-body and using the `application/json` media type. If the Wallet supports the Notification Endpoint, the Wallet MAY send one or more Notification Requests per Credential issued.
+The Wallet sends an HTTP POST request to the Notification Endpoint with the following parameters in the entity-body and using the `application/json` media type. If the Wallet supports the Notification Endpoint, the Wallet MAY send one or more Notification Requests per Credential issuance flow.
```

Comment:
I think the "notification_id` value received" is indeed better. 

I wallet may call multiple times the credential endpoint, using the same `access_token`. Thus, it can receive more than one `notification_id`.